runtime/compiler/compile/J9SymbolReferenceTable.cpp

```diff
@@ -462,7 +462,7 @@ J9::SymbolReferenceTable::findOrCreateVarHandleMethodTypeTableEntrySymbol(TR::Re
 
    TR::StaticSymbol *sym = TR::StaticSymbol::createMethodTypeTableEntry(trHeapMemory(),cpIndex);
    sym->setStaticAddress(entryLocation);
-   bool isUnresolved = *(j9object_t*)entryLocation == NULL;
+   bool isUnresolved = owningMethod->isUnresolvedVarHandleMethodTypeTableEntry(cpIndex);
```

Comment:
You could save one message by having `varHandleMethodTypeTableEntryAddress` return the 'unresolved' information as well.